components/07-layout/layout.njk

```diff
@@ -21,13 +21,13 @@
 
 <h2>Voorbeeld grid layout</h2>
 <section class="grid-example">
-  {{#each rows }}
+  {% for layoutrow in rows %}
     <div class="row">
-      {{#each columns }}
-        <div class="{{join (prefix this 'col-') ' ' }}">
-          <div class="col">{{join this ' ' }}</div>
+      {% for col in layoutrow.columns %}
```

Comment:
Waarom niet `for column in row.columns`..

Reply:
aangepast